client/modules/IDE/components/Sidebar.jsx

```diff
@@ -69,12 +69,12 @@ class Sidebar extends React.Component {
               <InlineSVG src={downArrowUrl} />
             </button>
             <ul className="sidebar__project-options">
-              <li>
+              <li className="sidebar_file">
```

Comment:
following [BEM](http://getbem.com/) conventions, i would name this class "sidebar__project-option"